### PR TITLE
don’t use sync ipc for publisher info

### PR DESCRIPTION
**Pull request by bridiver**
Submitter Checklist:

the diff is mostly an indenting change from wrapping in ipcRenderer.once

- [ ] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Test Plan:
1. Enable ledger
2. Visit some pubs long enough to add them
3. Pubs should show in ledger prefs

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by bsclifton** [comment]
@NejcZdovc would you be able to check this one out? 😄

File: js/constants/messages.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ const messages = {
   // Ledger
   LEDGER_PAYMENTS_PRESENT: _,
   LEDGER_PUBLISHER: _,
```
Comment (diracdeltas) [inline js/constants/messages.js]: i think you can remove this message and its handler in `app/ledger.js`

**Comment by diracdeltas** [comment]
seems like a bunch of payments tests are failing: https://travis-ci.org/brave/browser-laptop/jobs/243018061

i'm not really familiar with how this code works, so defer to @NejcZdovc

**Comment by bsclifton** [comment]
I just rebased- will watch tests and check out soon 😄

**Comment by bridiver** [comment]
@diracdeltas @bsclifton ledger appears to be broken in master. With a fresh profile I can't initialize ledger, but the content script appears to be functioning correctly in this PR and master

**Comment by bridiver** [comment]
actually it works now, I think my node modules were out-of-date

**Comment by bridiver** [comment]
there was a variable name capitalization mismatch. Ledger tests are working now

**Comment by bridiver** [comment]
there are automated tests. This doesn't add any new functionality

**Pull request merged**